### PR TITLE
Car details routes impl

### PR DESCRIPTION
**Pull request by alexansaa**
In this milestone, the following features have been implemented:
- React Route for car details
- This link isn't shown on the navigation bar
- You can navigate to that link using the "useNavigate" hook and pointing to "items/item_id" having item_id as the car id number
- Connected the cars details route to the car elements shown on the main page

### CODE REVIEW

**Review by ahmed025566 — APPROVED** [review]
Good job

**Review by ahmidou09 — APPROVED** [review]
Hi @alexansaa .

STATUS: APPROVED! 🥇 💯

Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

File: src/app.jsx [hunk 1]
```diff
@@ -31,19 +32,22 @@ const App = () => {
...
+        <Route path="/dashboard" element={<Dashboard />} />
+        <Route path="/reserve-form" element={<ReserveForm />} />
+        <Route path="/my-reservations" element={<MyReservations />} />
+        <Route path="/add-car-item" element={<AddCarItem />} />
+        <Route path="/delete-car-item" element={<DeleteCarItem />} />
+
+        <Route path="/items" element={<Outlet />}>
+          <Route path=":id" element={<CarDetail />} />
         </Route>
-      </Routes>
```
Comment (ahmidou09) [inline src/app.jsx:45]: I suggest you replace the code above with this:  
<Route path="/items/:id" element={<CarDetail />} />

**Review by ahmidou09 — APPROVED** [review]

**Review by Armynerh — APPROVED** [review]
APPROVED 
Hi @alexansaa ,
Your project is complete! There is nothing else to say other than... it's time to merge it 

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- well-written code
- Correct Linters

Optional suggestions
Every comment with the [optional] prefix won't stop the approval of this PR. However, I strongly recommend that you take them into account, as they can make your code better. 
- Try to fix the error suggested by the previous reviewer, as addressing it will really improve your application.

**Pull request merged**